dev.pulsar-edit.Pulsar.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id": "dev.pulsar-edit.Pulsar",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "22.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "pulsar",
+    "rename-desktop-file": "pulsar.desktop",
+    "rename-icon": "pulsar",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=ssh-auth",
+        "--share=network",
+        "--device=all",
+        "--filesystem=host",
```

Comment:
host is a very broad filesystem permission

Reply:
Generally I think there is one featurewhere Pulsar uses ssh (in an external package) but it should be fine.